### ISSUE TITLE
Nothing happens when attempting to create a table in an NW.js environment (when Tabulator has been "required")

### ISSUE THREAD

**Issue by NbrTDB** [olifolkerd/tabulator#3274]
**Describe the bug**
If Tabulator is loaded via require (either directly against the download JS or via a copy installed from NPM, e.g. `var Tabulator = require('tabulator-tables');`) then attempting to create a table with `var table = new Tabulator("#example-table", {})` seems to silently fail (no table appears, no changes to DOM & no error messages).

However, if tabulator is loaded via `<script>` tags pointed at the tabulator JS file, it works fine and creates the table.

I wondered if it's perhaps something to do with JS context (see a similarly affected project here: https://github.com/feross/drag-drop/issues/7#issuecomment-150975056)?

See also: https://docs.nwjs.io/en/latest/For%20Users/Advanced/JavaScript%20Contexts%20in%20NW.js/

**Tabulator Info**
v4.9.3

**Working Example**
Obviously a JSFiddle/Codepen won't be possible, but I have created a bare-minimum example project which can be downloaded below:

https://1drv.ms/u/s!Auotg3a5OkW09RaXS80_4eVvuWF_?e=m0E5xn

**To Reproduce**
A step by step guide to recreate the issue in your ~~JS Fiddle or Codepen~~ NW.js environment:
1. Open with `run.bat` (or via cmdline: `runtime/nw.exe ui/`
2. Press F12 to open devtools
3. Run the test function in the console: `initialiseWorkTable()`
4. Table should appear, but nothing happens
5. Try commenting out the `require` in `test.js` and relying on the `script` tags instead; this time it should work

**Expected behavior**
I would expect the table to appear regardless of whether `script` tags or `require` was used to load the Tabulator library.

**Desktop (please complete the following information):**
 - OS: Windows 10 20H2
 - NW.js (v0.50.1): 
 ```
ares: "1.16.1"
brotli: "1.0.9"
chromium: "87.0.4280.66"
icu: "67.1"
llhttp: "2.1.3"
modules: "88"
napi: "7"
nghttp2: "1.41.0"
node: "15.3.0"
node-webkit: "0.50.1"
nw: "0.50.1"
nw-commit-id: "2d5b629-518f361-e10550e-d168d6d"
nw-flavor: "sdk"
openssl: "1.1.1g"
unicode: "13.0"
uv: "1.40.0"
v8: "8.7.220.25"
zlib: "1.2.11"
```

**Additional context**
As an aside (and in case it helps), in my actual application I observe a _different_ behaviour, but I suspect this is because my app launches with a splash window and then opens a secondary "main" window thereafter (which probably does something to the JavaScript context as well...). In this case it's like it can't see `window` or `document` exist at all.

```
(unknown) Uncaught ReferenceError: window is not defined
    at Tabulator.initializeElement (C:\...\ui\js\lib\tabulator-master\dist\js\tabulator.js:9351:19)
    at new Tabulator (C:\...\ui\js\lib\tabulator-master\dist\js\tabulator.js:8738:12)
```

However, once again, in my actual application if I use `<script>` tags in the HTML of the main window, then Tabulator seems to create the table without issues.

**Comment by olifolkerd** [comment]
You probably need to import rather than require

**Comment by NbrTDB** [comment]
Thanks for the reply, I appreciate it. I had a go, but `import` causes errors about trying to import outside a module.

However, I have managed to get it working in a way besides hardcoded <script> tags; in fact two ways seem to work:

1) Create a script element with its source set to the file path and add that to the document (functionally the same as the script tag, really, but without being embedded in HTML)

2) Compile to binary with nwjc and evaluate it using `evalNWBin()`

Both of the above approaches work fine in the test environment and also my actual use-case, and work well enough for me.

Thinking about it, I'm not sure if NW.js support (or lack thereof?) really counts as a "bug" anyway? I suppose this would be more of a feature request? In any case I got it working so I'll close this out. Thanks for your time!"
